org.multimc.MultiMC.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.multimc.MultiMC
+default-branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15'
+command: multimc
+separate-locales: false
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
I guess it doesn't actually work natively on Wayland?

Reply:
Which part of the program isn't working natively in Wayland?